### PR TITLE
Teshari additions

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
Proof I don't go halfsies on anything

I spent like fifteen minutes writing a long descriptive PR for this but then my power went out and I lost it so all you get is the changelog

Recoloured Tesh ear/feather/whatever sprites from Haven-13/Haven-Urist#30 who in turn got the sprites from Polaris and Arkstation.

## Why It's Good For The Game
Previously Teshari were embarrassingly non-unique, so I decided to finally bite the bullet and add quite a bit more in terms of differentiating.

## Changelog
:cl:
add: You can now pick up Teshari into mob holders. Teshari and other holdable mobs cannot pick up other holdable mobs to prevent recursion of any kind.
add: Two new coloured customization options for Tesh, including head feathers/ears and optional body colour.
add: Teshari now have a unique language (with no unique sprite as of yet) called Schechi.
add: You can now only put <b>one</b> mob holder into a duffel bag, and no other bags.
fix: Teshari actually have eyes now
fix: You can no longer eat some simplemobs alive (if you know, you know)
fix: mob holders can now breathe in bags
tweak: Minor Teshari diet adjustment, they now prefer meat and dislike grain/gross foods.
tweak: Teshari blood is no longer ammonia.
/:cl:

### CODE REVIEW

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: ca83a63d9154e7bd47271dd9f5ce5d44eb19d7af
Pull Request: d8e5608f0d614f97f871563e68fa31a10f7701ca
Server: 52f18b307da9ed9bafb2f173c5db9806fde13a70

File: code/modules/mob/living/carbon/human/species_types/teshari.dm [hunk 1]
```diff
@@ -2,16 +2,22 @@
...
 	id = "teshari"
 	default_color = "6060FF"
-	species_traits = list(MUTCOLORS, NOEYESPRITES, NO_UNDERWEAR)
-	default_features = list("mcolor" = "0F0", "wings" = "None")
-	disliked_food = GROSS | RAW
-	liked_food = JUNKFOOD | FRIED
+	species_traits = list(MUTCOLORS, EYECOLOR, NO_UNDERWEAR)
+	mutant_bodyparts = list("teshari_feathers", "teshari_body_feathers")
+	default_features = list("mcolor" = "0F0", "wings" = "None", "teshari_feathers" = "Plain", "teshari_body_feathers" = "Plain")
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/chicken
```
Comment (ZephyrTFA) [inline code/modules/mob/living/carbon/human/species_types/teshari.dm:8]: Heh

**Comment by swifterjack** [comment]
nooooo no more eating ian for infinite food as a moth

**Comment by super12pl** [comment]
image

**Pull request merged**